inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/extract/IcebergExtractNode.java

```diff
@@ -110,7 +110,10 @@ public Map<String, String> tableOptions() {
         options.put(IcebergConstant.TABLE_KEY, tableName);
         options.put(IcebergConstant.CATALOG_TYPE_KEY, catalogType.name());
         options.put(IcebergConstant.CATALOG_NAME_KEY, catalogName);
+        // support streaming only
         options.put(IcebergConstant.STREAMING, "true");
+        options.put(IcebergConstant.STARTING_STRATEGY_KEY,
```

Comment:
Guess we need this parameter to be configurable, use FROM_LATEST by default and stili we support changing it